acceptance/epinio_installation_test.go

```diff
@@ -28,4 +30,28 @@ var _ = Describe("Epinio Installation", func() {
 		Expect(out).To(MatchRegexp("linkerd-sp-validator"))
 		Expect(out).To(MatchRegexp("linkerd-destination"))
 	})
+
+	It("redirects http to https", func() {
+		request, err := http.NewRequest(
+			"GET",
+			strings.Replace(fmt.Sprintf("%s/info", serverURL), "https", "http", 1),
+			strings.NewReader(""))
+
+		Expect(err).ToNot(HaveOccurred())
+		request.SetBasicAuth(env.EpinioUser, env.EpinioPassword)
+		client := env.Client()
+
+		// Don't follow redirects
+		client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		}
+
+		r, err := client.Do(request)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(r.StatusCode).To(Equal(http.StatusMovedPermanently))
+		location, err := r.Location()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(location.String()).To(Equal(fmt.Sprintf("%s/info", serverURL)))
```

Comment:
The `fmt.Sprintf("%s/info", serverURL)` is used twice (here and in line 37).
Maybe make it a variable at the top and share?

Reply:
done (amended)

Reply:
I do not see the variable. Only that the PR was rebased to newer main.